test/converter/converter_cases.py

```diff
@@ -142,14 +169,119 @@ def forward(self, x):
         x = self.linear(x)
         x1 = self.linear1(x)
         x2 = self.linear2(x)
-        x = x1 + x2
+        x = x1 + x2  # built-in method add
         x = self.relu(x)
+        x = x.add(x2)  # method add
         return x
 
     def input_fn(self) -> Tensor:
         return rand(3, self.in_dim)
 
 
+class _Permute(ConverterModule):
+    def __init__(self):
+        super().__init__()
+        self.in_dim = (5, 3)
+        out_dim = 2
+        self.linear = Linear(self.in_dim[-1], out_dim)
+        self.linear2 = Linear(self.in_dim[-2], out_dim)
+
+    def forward(self, x):
+        x = self.linear(x)
+        x = x.permute(0, 2, 1)  # method permute
+        x = self.linear2(x)
+        x = permute(x, (0, 2, 1))  # function permute
+        return x
+
+    def input_fn(self) -> Tensor:
+        return rand(3, *self.in_dim)
+
+    def loss_fn(self) -> Module:
+        return CrossEntropyLoss()
+
+
+class _Transpose(ConverterModule):
+    def __init__(self):
+        super().__init__()
+        self.in_dim = (5, 3)
+        out_dim = 2
+        out_dim2 = 3
+        self.linear = Linear(self.in_dim[-1], out_dim)
+        self.linear2 = Linear(self.in_dim[-2], out_dim2)
+
+    def forward(self, x):
+        x = self.linear(x)
+        x = x.transpose(1, 2)  # method transpose
+        x = self.linear2(x)
+        x = transpose(x, 1, 2)  # function transpose
+        return x
+
+    def input_fn(self) -> Tensor:
+        return rand(3, *self.in_dim)
+
+    def loss_fn(self) -> Module:
+        return CrossEntropyLoss()
+
+
+class _TolstoiCharRNN(ConverterModule):
+    def __init__(self):
+        super(_TolstoiCharRNN, self).__init__()
+        self.hidden_dim = 64
+        self.num_layers = 2
+        self.seq_len = 15
+        self.vocab_size = 25
+
+        self.embedding = Embedding(
+            num_embeddings=self.vocab_size, embedding_dim=self.hidden_dim
+        )
+        self.dropout = Dropout(p=0.2)
+        self.lstm = LSTM(
+            input_size=self.hidden_dim,
+            hidden_size=self.hidden_dim,
+            num_layers=self.num_layers,
+            dropout=0.36,
+            batch_first=True,
+        )
+        self.lstm.bias_ih_l0.data = zeros_like(self.lstm.bias_ih_l0)
+        self.lstm.bias_ih_l1.data = zeros_like(self.lstm.bias_ih_l1)
+        self.lstm.bias_ih_l0.requires_grad = False
+        self.lstm.bias_ih_l1.requires_grad = False
```

Comment:
Is there a specific reason why the bias terms are not differentiable?

Reply:
Yes, because this is the implementation in DeepOBS.
In DeepOBS, the tensorflow version of Tolstoi Char RNN does have only one bias.
The default LSTM in PyTorch has two redundant biases.
After discussion with Frank, we decided to deactivate the unnecessary bias.

Reply:
Thanks.